### PR TITLE
test: default loader name and plugin name

### PR DESCRIPTION
**Pull request by rishabh3112**
**What kind of change does this PR introduce?**
feat

**Did you add tests for your changes?**
Not needed, already present

**If relevant, did you update the documentation?**
No

**Summary**
Enforced use of process.cwd() as default `output-path`

**Does this PR introduce a breaking change?**
No

**Other information**
Part of Roadmap

### CODE REVIEW

**Review by jamesgeorge007 — COMMENTED** [review]
`yeoman-environment` handles it already.
https://github.com/yeoman/environment/blob/main/lib/environment.js#L223

**Comment by alexander-akait** [comment]
@jamesgeorge007 Good catch, maybe we just need add test(s)

**Comment by rishabh3112** [comment]
> Good catch, maybe we just need add test(s)

Tests are already present. Why this was included in roadmap?

**Comment by alexander-akait** [comment]
Can you provide link on tests? Maybe I something missed

**Comment by rishabh3112** [comment]
> Can you provide link on tests? Maybe I something missed

`output-path` is not provided in all the tests below:

https://github.com/webpack/webpack-cli/blob/master/test/loader/loader.test.js#L22
https://github.com/webpack/webpack-cli/blob/master/test/loader/loader.test.js#L29

https://github.com/webpack/webpack-cli/blob/master/test/plugin/plugin.test.js#L22
https://github.com/webpack/webpack-cli/blob/master/test/plugin/plugin.test.js#L29

**Comment by alexander-akait** [comment]
No tests, https://github.com/webpack/webpack-cli/blob/master/test/loader/loader.test.js#L22 is not enough for testing, it is stupid test on questions, it should be fully test with checking on files https://github.com/webpack/webpack-cli/blob/master/test/loader/loader.test.js#L43

**Comment by rishabh3112** [comment]
Line 29 has it.

**Comment by alexander-akait** [comment]
Line 29 pass `loaderName` - `let { stdout } = await runPromptWithAnswers(__dirname, ['loader'], [`${loaderName}${ENTER}`]);`, we should have test without passing loader name (full test)

**Comment by rishabh3112** [comment]
Loader name is not being passed as `output-path`, it is an answer to an inquirer question asked by generator.

Nevertheless will add test for default loader name as well .

File: packages/generators/src/index.ts [hunk 1]
```diff
@@ -18,6 +18,9 @@ class GeneratorsCommand {
             },
             [],
             async (outputPath) => {
```
Comment (anshumanv) [inline packages/generators/src/index.ts:20]: can just do default param?

```suggestion
            async (outputPath = process.cwd()) => {
```

Comment (rishabh3112) [reply packages/generators/src/index.ts:20]: Yup we can but I personally don't like calling functions as default parameter 😄, looks hacky to me.

**Comment by alexander-akait** [comment]
Thanks

**Pull request merged**